src/decision_process/examples.md

```diff
@@ -0,0 +1,297 @@
+> Rust!
+
+This indicates that they would like to merge the proposal. At the moment, there
+is no decision pending, so rustbot would add a comment that looks like the
+following:
+
+> Hello! @Alan has proposed to merge this. This is a **reversible decision**,
+> which means that it will be affirmed once the "final comment period" of 10
+> days have passed, unless a team member places a "hold" on the decision (or
+> cancels it).
+>
+> | Team member | State |
+> | --- | --- |
+> | @Alan | **merge** |
+> | @Barbara | |
+> | @Grace | |
+> | @Niklaus | |
+
+As the comment says, the PR is now in "pending decision" state, with Alan
+having kicked off the process with a proposal to merge. Alan's status of
+**merge** will link to his comment.
+
+Now, for this particular proposal, Barbara has a concern. She thinks that the
+proposal has overlooked an important consideration. She writes a comment:
+
+> @rustbot hold
+>
+> Did you consider reversing the polarity? Or the impact on the flux capacitor?
+
+At this point, rustbot updates the state:
+
+> Hello! @Alan has proposed to merge this. This is a **reversible decision**,
+> which means that it will be affirmed once the "final comment period" of 10
+> days have passed, unless a team member pauses or suspends the decision.
+>
+> | Team member | State |
+> | --- | --- |
+> | @Alan | **merge** |
+> | @Barbara | **hold** |
+> | @Grace | |
+> | @Niklaus | |
+
+Alan is currently busy at work, though, so by by the time that he and Barbara
+get a chance to talk, 11 days have passed. (Alan and Barbara receive a ping
+from rustbot after a week or so.) Once they get a chance to talk, Alan
+fully addresses Barbara's concern, so Barbara posts:
+
+> @rustbot merge
+
+The comment is updated:
+
+> Hello! @Alan has proposed to merge this. This is a **reversible decision**,
+> which means that it will be affirmed once the "final comment period" of 10
+> days have passed, unless a team member pauses or suspends the decision.
+>
+> | Team member | State |
+> | --- | --- |
+> | @Alan | **merge** |
+> | @Barbara | ~~hold~~ **merge** |
+> | @Grace | |
+> | @Niklaus | |
+
+Barbara's previous ~~hold~~ status links to her previous comment setting her
+status to `hold`, and her current **merge** status links to her more recent
+comment setting her status to `merge`.
+
+At this point, all the statuses are either empty, **abstain**, or **merge**,
+and more than 10 days have passed since the FCP started. Therefore, it
+completes immediately.
+
+## Authoring an RFC (illustration of `rustbot restart`)
+
+After some time, the proposal is completed and an RFC is proposed. This is a
+reversible decision. Alan, as the liaison, proposes to merge the RFC with
+`@rustbot merge`, and the decision making process proceeds as above.
+
+This time, Niklaus has a concern:
+
+> @rustbot hold
+>
+> I have not had time to read this yet! Give me a bit of time to write it up.
+
+After 7 days have passed, rustbot writes to her:
```

Comment:
I think the "cast of characters" text has established that Niklaus identifies as male.

```suggestion
After 7 days have passed, rustbot writes to him:
```